application/src/main/java/be/jpendel/application/SignupUC.java

```diff
@@ -11,6 +11,8 @@ public SignupUC(MemberRepository memberRepository) {
         this.memberRepository = memberRepository;
     }
 
+    //TODO brrrr 'new Member' ipv een builder
+    // Volgens mijn begrip mag de applicatie laag niet een Member rechtreeks aanmaken ..waar is CreateMemberCommand ?
```

Comment:
Moet de UC niet weten 'dat' het moet gebeuren en niet het 'hoe' ? tis maar een bedenking.. (voor mij is 'new' een vies woord ;))